1-exercises/A-reading-list/script.js

```diff
@@ -1,6 +1,30 @@
+let content = document.querySelector('#content');
 function readingList(books) {
   // Write your code here...
-}
+  let unorderedList = document.createElement("ul"); // created unordered list tag
+  content.append(unorderedList);
+    
+  //iterate through the list of books
+   bookList = books.forEach(book => {
+    paraElement = document.createElement("p"); //p element
```

Comment:
When creating new variables, need to use the keywords let or const depending on the variable. There should always be a keyword before the variable name.